model/ravelWrap.cc

```diff
@@ -439,7 +449,13 @@ namespace
   /// @throw if type does not match global dimension type
   void Ravel::setDimension(Dimension::Type type,const std::string& units)
   {
-    auto descr=description();
+    return setDimension(selectedHandle(), type, units);
+  }
+
+  /// @throw if type does not match global dimension type
```

Comment:
Minor thing - it is not necessary to duplicate the doxygen comment from the declaration to definition.